further_link/endpoint/upload.py

```diff
@@ -47,3 +53,31 @@ async def upload(request):
     return web.Response(
         text=json.dumps({"success": True, "fetched_urls": fetched_urls})
     )
+
+
+async def bluetooth_upload(
+    device, uuid: str, message: bytearray, characteristic_to_report_on: str
+):
+    device.write_value("IN_PROGRESS", characteristic_to_report_on)
```

Comment:
keep this only if necessary, would be nice to remove if poss. if we do keep it maybe it could be a json for consistency.